<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsColorRampShaderWidgetBase</class>
 <widget class="QWidget" name="QgsColorRampShaderWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="4" column="0" colspan="3">
    <widget class="QTreeWidget" name="mColormapTreeWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <attribute name="headerMinimumSectionSize">
      <number>10</number>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>70</number>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Color</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Label</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QgsColorRampButton" name="btnColorRamp">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="mColorInterpolationLabel">
     <property name="text">
      <string>Interpolation</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="mClassificationModeLabel">
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mClassificationModeComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="mNumberOfEntriesLabel">
       <property name="text">
        <string>Classes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsSpinBox" name="mNumberOfEntriesSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="mColorInterpolationComboBox"/>
   </item>
   <item row="6" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="mClassifyButton">
       <property name="text">
        <string>Classify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mAddEntryButton">
       <property name="toolTip">
        <string>Add values manually</string>
       </property>
       <property name="icon">
        <iconset resource="../../images/images.qrc">
         <normaloff>:/images/themes/default/symbologyAdd.svg</normaloff>:/images/themes/default/symbologyAdd.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mDeleteEntryButton">
       <property name="toolTip">
        <string>Remove selected row(s)</string>
       </property>
       <property name="icon">
        <iconset resource="../../images/images.qrc">
         <normaloff>:/images/themes/default/symbologyRemove.svg</normaloff>:/images/themes/default/symbologyRemove.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mLoadFromBandButton">
       <property name="toolTip">
        <string>Load color map from band</string>
       </property>
       <property name="icon">
        <iconset resource="../../images/images.qrc">
         <normaloff>:/images/themes/default/mActionReload.svg</normaloff>:/images/themes/default/mActionReload.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mLoadFromFileButton">
       <property name="toolTip">
        <string>Load color map from file</string>
       </property>
       <property name="icon">
        <iconset resource="../../images/images.qrc">
         <normaloff>:/images/themes/default/mActionFileOpen.svg</normaloff>:/images/themes/default/mActionFileOpen.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mExportToFileButton">
       <property name="toolTip">
        <string>Export color map to file</string>
       </property>
       <property name="icon">
        <iconset resource="../../images/images.qrc">
         <normaloff>:/images/themes/default/mActionFileSaveAs.svg</normaloff>:/images/themes/default/mActionFileSaveAs.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>48</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mLegendSettingsButton">
       <property name="text">
        <string>Legend Settingsâ€¦</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QCheckBox" name="mClipCheckBox">
     <property name="toolTip">
      <string>If checked, any pixels with a value out of range will not be rendered</string>
     </property>
     <property name="text">
      <string>Clip out of range values</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="mUnitLineEdit">
     <property name="toolTip">
      <string>Unit suffix</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mUnitLabel">
     <property name="text">
      <string>Label unit suffix</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mColorInterpolationLabel_2">
     <property name="text">
      <string>Color ramp</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mLabelPrecisionLabel">
     <property name="text">
      <string>Label precision</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QgsSpinBox" name="mLabelPrecisionSpinBox">
     <property name="toolTip">
      <string>Precision of upper and lower values in label text.
Positive is number of decimal places
Negative rounds to powers of 10</string>
     </property>
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsColorRampButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorrampbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mColorInterpolationComboBox</tabstop>
  <tabstop>btnColorRamp</tabstop>
  <tabstop>mUnitLineEdit</tabstop>
  <tabstop>mLabelPrecisionSpinBox</tabstop>
  <tabstop>mColormapTreeWidget</tabstop>
  <tabstop>mClassificationModeComboBox</tabstop>
  <tabstop>mNumberOfEntriesSpinBox</tabstop>
  <tabstop>mClassifyButton</tabstop>
  <tabstop>mAddEntryButton</tabstop>
  <tabstop>mDeleteEntryButton</tabstop>
  <tabstop>mLoadFromBandButton</tabstop>
  <tabstop>mLoadFromFileButton</tabstop>
  <tabstop>mExportToFileButton</tabstop>
  <tabstop>mLegendSettingsButton</tabstop>
  <tabstop>mClipCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
